app/models/import_batch.rb

```diff
@@ -30,6 +30,11 @@ class ImportBatch < MappingsBatch
     validator: HostInWhitelistValidator,
     message: I18n.t('mappings.bulk.new_url_must_be_on_whitelist')
   }
+  validates :archive_urls, each_in_collection: {
+    validator: LengthValidator,
+    maximum: (64.kilobytes - 1),
+    message: I18n.t('mappings.new_url_too_long')
```

Comment:
This message is [specific to new URL](https://github.com/alphagov/transition/blob/4f2fe54d6753e4fcff11390c7ba8f91a75a1665e/config/locales/en.yml#L27), which could be confusing to users. We could make that message more generic, add a separate one for custom archive URLs or make the message take the name of the field to use.